src/utils/IdealGraphVisualizer/Coordinator/src/main/java/com/sun/hotspot/igv/coordinator/GraphNode.java

```diff
@@ -59,6 +60,20 @@ public GraphNode(InputGraph graph) {
         this(graph, new InstanceContent());
     }
 
+    private boolean selected = false;
```

Comment:
You should move this field up to the other field declarations.

Reply:
done